crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -209,6 +219,48 @@ It can greatly improve performance (and readability) in such situations as it pr
         }
     }
 
+    /// Choose the default visualizers to enable for this entity.
+    fn choose_default_visualizers(
+        &self,
+        entity_path: &EntityPath,
+        visualizable_entities_per_visualizer: &PerVisualizer<VisualizableEntities>,
+        indicated_entities_per_visualizer: &PerVisualizer<IndicatedEntities>,
+    ) -> SmallVisualizerSet {
+        let available_visualizers: HashSet<&ViewSystemIdentifier> =
+            visualizable_entities_per_visualizer
+                .iter()
+                .filter_map(|(visualizer, ents)| {
+                    if ents.contains(entity_path) {
+                        Some(visualizer)
+                    } else {
+                        None
+                    }
+                })
+                .collect();
+
+        let mut visualizers: SmallVisualizerSet = available_visualizers
+            .iter()
+            .filter_map(|visualizer| {
+                if indicated_entities_per_visualizer
+                    .get(*visualizer)
+                    .map_or(false, |matching_list| matching_list.contains(entity_path))
+                {
+                    Some(**visualizer)
+                } else {
+                    None
+                }
+            })
+            .collect();
+
+        // If there were no other visualizers, but the SeriesLineSystem is available, use it.
+        if visualizers.is_empty() && available_visualizers.contains(&SeriesLineSystem::identifier())
+        {
+            visualizers.insert(0, SeriesLineSystem::identifier());
+        }
```

Comment:
unfortunate that we can't just call into the base for the rest (don't see how either). But otherwise perfect, just as planned!